core/src/Instance/AadAuthority.cs

```diff
@@ -35,63 +35,51 @@ namespace Microsoft.Identity.Core.Instance
 {
     internal class AadAuthority : Authority
     {
-        private const string AadInstanceDiscoveryEndpoint = "https://login.microsoftonline.com/common/discovery/instance";
-        private static readonly HashSet<string> TrustedHostList = new HashSet<string>()
+        internal static readonly HashSet<string> TrustedHostList = new HashSet<string>()
         {
-            "login.windows.net", // Microsoft Azure Worldwide
+            "login.windows.net", // Microsoft Azure Worldwide - Used in validation scenarios where host is not this list 
             "login.chinacloudapi.cn", // Microsoft Azure China
-            "login.cloudgovapi.us", // Microsoft Azure US Government
+            "login.microsoftonline.de", // Microsoft Azure Blackforest
+            "login-us.microsoftonline.com", // Microsoft Azure US Government - Legacy
             "login.microsoftonline.us", // Microsoft Azure US Government
             "login.microsoftonline.com", // Microsoft Azure Worldwide
-            "login.microsoftonline.de" // Microsoft Azure Blackforest
+            "login.cloudgovapi.us" // Microsoft Azure US Government
         };
 
-        public AadAuthority(string authority, bool validateAuthority) : base(authority, validateAuthority)
+        public const string DefaultTrustedAuthority = "login.microsoftonline.com";
```

Comment:
Inconsistent naming - TrustedHostList  / DefaultTrustedAuthority  - also inconsistent visibility - please make both public as the class is internal.

Reply:
good point it is host not authority, changed to host